### DISCUSSION TITLE
Welcome to fonttools Discussions!

### DISCUSSION THREAD

**Discussion by anthrotype** [fonttools/fonttools#2123] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anthrotype** [comment]
(that was the original template message)
I'm actually not sure what we want to do with this new forum, but I was simply curious to try it out so I have enabled the feature.
I guess we could retire the old https://groups.google.com/d/forum/fonttools and use this instead. The latter has very low traffic anyway.

**Reply by belluzj** [reply]
There's also the Gitter chatroom which could be moved here as well?

**Comment by anthrotype** [comment]
> Gitter

Yeah maybe, even though that's more for real-time communication, while AFAIU this one looks more like a forum. OK, we don't often engage in real-time communication on that gitter chat either.

**Reply by anthrotype** [reply]
(I should have replied inside the inline reply box below your message.. still learning :)

**Comment by ctrlcctrlv** [comment]
Yup I use fonttools and like it

I used it to trick my users into thinking I actually made a release recently lol

https://github.com/ctrlcctrlv/TT2020/blob/master/build/version_0.2_patch.py

**Comment by BoldMonday** [comment]
Long time user of FontTools here. It is an indispensable part of our production system where it is used predominantly to enhance generated binaries. And we use the subsetter too.

I'm mainly following this repo to stay informed of the latest updates and tricks.

**Comment by typoman** [comment]
I'm a big fan of fontTools! I use fontTools to fix the issues in my fonts. I even made a JIT font compiler using ufo2ft and fontTools, so I just lazy compile only the changing contours without having to compile the whole font every time I change a contour. 

I also come here to stay informed about changes and new features. Thank you all for maintaining this magical library!

**Reply by behdad** [reply]
Thanks Bahman!

The incremental build is something related to what we are discussing in:

  https://github.com/fonttools/fonttools/issues/1095

I'll comment on incremental build there soon.

Cheers,
b

**Comment by ftCLI** [comment]
Hi all,

I released a small command line interface for FontTools. Still lacks documentation, but has decent help.

It helps making small fixes as well as correctly compiling the name table.

https://github.com/ftCLI/ftCLI

**Reply by behdad** [reply]
Hi. Fonttools itself also has a CLI. It might be less confusing if you name your project something else. But otherwise, cool stuff. Thanks for sharing.

**Reply by ftCLI** [reply]
Hi behdad,

thanks. I renamed it as pyFontEditor.

**Reply by behdad** [reply]
nice. :)

**Reply by alerque** [reply]
Ouch, it looks like you re-initialized and org and repository instead of using GitHub's rename features. Hence old links above are broken, the new one as far as I can find is: https://github.com/ftCLI/ftCLI

Additionally, I can't find any evidence of the above mentioned rename to `pyFontEditor`. In fact the only thing I can turn up in a search is this conversation. Can you link to where the canonical source for the project is now? I'd like to update Arch AUR packages with the correct name but can't find the project.

**Reply by ftCLI** [reply]
Sorry, missed this message.

It has been renamed to ftCLI and the repository is https://github.com/ftCLI/ftCLI
